documentation/modules/mtv-resources-and-services.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
 
+:_content-type: REFERENCE
```

Comment:
@sbeskin-redhat This looks like a concept module to me. I'd read it to understand about customer resources and services. I don't know if I'd refer back to this as a reference.